Cargo.toml

```diff
@@ -1,76 +1,91 @@
 [package]
 name = "swagger"
-version = "6.5.0"
+version = "7.0.0-rc1"
 authors = ["Metaswitch Networks Ltd"]
 license = "Apache-2.0"
 description = "A set of common utilities for Rust code generated by OpenAPI Generator"
 homepage = "https://github.com/Metaswitch/swagger-rs"
 repository = "https://github.com/Metaswitch/swagger-rs"
 readme = "README.md"
 keywords = ["swagger"]
-edition = "2018"
+edition = "2021"
 
 [badges.travis-ci]
 repository = "Metaswitch/swagger-rs"
 
 [features]
```

Comment:
you've removed features from lib.rs documentation that still exist here - please resolve